### ISSUE TITLE
google_storage_project_service_account does not have attribute 'email_address'

### ISSUE THREAD

**Issue by timReynolds** [terraform-providers/terraform-provider-google#1954]
### Terraform Version
Terraform v0.11.8
+ provider.google v1.16.2

### Affected Resource(s)
Please list the resources as a list, for example:
- google_storage_project_service_account

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
resource "google_pubsub_topic" "gcs-pubsub-topic" {
  name    = "gcs-uploads"
  project = "${module.project.project_id}"
}

data "google_storage_project_service_account" "gcs_account" {
  project = "${module.project.project_id}"
}

resource "google_pubsub_topic_iam_binding" "binding" {
  topic   = "${google_pubsub_topic.gcs-pubsub-topic.name}"
  project = "${module.project.project_id}"
  role    = "roles/pubsub.publisher"
  members = ["serviceAccount:${data.google_storage_project_service_account.gcs_account.email_address}"]
}
```

### Debug Output
```shell
Error: Error running plan: 1 error(s) occurred:

* google_pubsub_topic_iam_binding.binding: 1 error(s) occurred:

* google_pubsub_topic_iam_binding.binding: Resource 'data.google_storage_project_service_account.gcs_account' does not have attribute 'email_address' for variable 'data.google_storage_project_service_account.gcs_account.email_address'
```

### Expected Behavior
Service account should be located and permission bound.

### Actual Behavior
Error 

### Steps to Reproduce
1. `terraform plan`

Please let me know if I can provide any additional information to help identify the problem.

**Comment by vishen** [comment]
This looks to be fixed by this PR - https://github.com/terraform-providers/terraform-provider-google/pull/1913. Presumably this will be fixed in the next release.

**Comment by paddycarver** [comment]
Yup! This should be addressed in the next release.

**Comment by rileykarson** [comment]
Hey @timReynolds! Sorry for not seeing this until now!

We released [version 1.17 of the Google Provider](https://github.com/terraform-providers/terraform-provider-google/blob/master/CHANGELOG.md#1170-august-22-2018) on August 22nd, so the field you were trying to use *was* released... just only on a newer version of the provider. 

Unfortunately, the docs at terraform.io only reflect the most recent release of the provider. My best recommendation for you here is to update to 1.17.1. We follow semantic versioning, so without a major version change all of our changes should be backwards compatible!

If you have a really really good reason to stay on 1.16.2, you can use `id` instead of `email_address` but please add a TODO to get off of it once you do upgrade - `id` is an internal undocumented field that can be changed at any time that we... accidentally used to have documented here.

I'm going to close out this issue but if this doesn't work after upgrading or you have any other thoughts, feel free to reopen the issue!
